intermediate_source/torch_export_nightly_tutorial.rst

```diff
@@ -0,0 +1,728 @@
+different inputs. Note that while the output ``ExportedGraph`` is callable and can be
+called in the same way as the original input callable, it is not a ``torch.nn.Module``.
+We will detail the ``dynamic_shapes`` argument later in the tutorial.
+
+.. code-block:: python
+
+    import torch
+    from torch.export import export
+
+    class MyModule(torch.nn.Module):
+        def __init__(self):
+            super().__init__()
+            self.lin = torch.nn.Linear(100, 10)
+
+        def forward(self, x, y):
+            return torch.nn.functional.relu(self.lin(x + y), inplace=True)
+
+    mod = MyModule()
+    exported_mod = export(mod, (torch.randn(8, 100), torch.randn(8, 100)))
+    print(type(exported_mod))
+    print(exported_mod(torch.randn(8, 100), torch.randn(8, 100)))
+
+.. code-block:: bash
+
+    <class 'torch.export.exported_program.ExportedProgram'>
+    tensor([[0.0000, 1.2178, 0.0000, 0.4397, 0.4774, 0.0000, 0.0000, 0.0943, 0.0000,
+            0.4656],
+            [0.8333, 0.0000, 0.5912, 0.0000, 1.4689, 0.2122, 0.1996, 0.4628, 0.0000,
+            0.7495],
+            [0.0000, 0.0000, 0.3900, 0.0000, 0.0000, 0.0000, 0.4515, 0.0000, 0.8187,
+            0.8938],
+            [0.5753, 0.7709, 0.0000, 0.0000, 0.0000, 0.8081, 0.0000, 0.0000, 0.8002,
+            0.9441],
+            [0.0000, 0.0000, 0.0000, 0.0000, 0.5711, 1.0921, 0.3438, 0.3268, 0.4640,
+            0.0000],
+            [0.0000, 0.0000, 0.0000, 0.2434, 0.7253, 0.6886, 0.0000, 0.6982, 0.5100,
+            0.0000],
+            [0.2279, 0.0000, 1.2951, 1.1055, 0.0000, 0.0000, 0.0000, 0.2088, 0.0000,
+            0.5022],
+            [0.0000, 0.0000, 1.1468, 0.0000, 0.5220, 1.1592, 0.9096, 0.0000, 0.4248,
+            1.2142]], grad_fn=<ReluBackward0>)
+
+Let's review some attributes of ``ExportedProgram`` that are of interest.
+
+The ``graph`` attribute is an `FX graph <https://pytorch.org/docs/stable/fx.html#torch.fx.Graph>`__
+traced from the function we exported, that is, the computation graph of all PyTorch operations.
+The FX graph has some important properties:
+
+- The operations are "ATen-level" operations.
+- The graph is "functionalized", meaning that no operations are mutations.
+
+The ``graph_module`` attribute is the ``GraphModule`` that wraps the ``graph`` attribute
+so that it can be ran as a ``torch.nn.Module``.
+
+.. code-block:: python
+
+    print(exported_mod)
+    print(exported_mod.graph_module)
+
+.. code-block:: bash
+
+    ExportedProgram:
+        class GraphModule(torch.nn.Module):
+            def forward(self, arg0_1: f32[10, 100], arg1_1: f32[10], arg2_1: f32[8, 100], arg3_1: f32[8, 100]):
+                # File: torch_export_nightly_tutorial.py:69, code: return torch.nn.functional.relu(self.lin(x + y), inplace=True)
+                add: f32[8, 100] = torch.ops.aten.add.Tensor(arg2_1, arg3_1);  arg2_1 = arg3_1 = None
+                permute: f32[100, 10] = torch.ops.aten.permute.default(arg0_1, [1, 0]);  arg0_1 = None
```

Comment:
this does not seem correct. it should be aten.t.default -- you might need to rebase